### PR TITLE
docs: update node snippets version to 4

### PR DESCRIPTION
**Pull request by JenniferMah**
Jira ticket: https://issues.corp.twilio.com/browse/DI-1470
Update hand-crafted node snippets to version 4 for MVR.

### CODE REVIEW

**Comment by childish-sambino** [comment]
Also need to change some of the TaskRouter examples that looks like this:
```ts
client.taskrouter.v1
  .workspaces(workspaceSid)
  .workers()
  .cumulativeStatistics()
```

Check the v4 upgrade guide for the list of ones that changed.

**Comment by JenniferMah** [comment]
It looks like the snippet for Real-Time Workers Statistics already [reflects the change](https://github.com/TwilioDevEd/api-snippets/blob/DI-1470/rest/taskrouter/statistics/workers/realtime/example-1.4.x.js#L11) but I've updated the snippet that includes the cumulativeStatistics workers.

**Review by childish-sambino — APPROVED** [review]

File: rest/taskrouter/statistics/workers/cumulative/example-1.4.x.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const client = require('twilio')(accountSid, authToken);
 
 client.taskrouter.v1
   .workspaces(workspaceSid)
-  .workers()
+  .workers
```
Comment (bermudezmt) [inline rest/taskrouter/statistics/workers/cumulative/example-1.4.x.js:11]: Hi @JenniferMah , the 1.3.x example where we use `workers()` or `workers('WKXXX')` works for me when testing but not this one. Is there a reason why that is no longer required in 1.4?

Comment (JenniferMah) [reply rest/taskrouter/statistics/workers/cumulative/example-1.4.x.js:11]: Hi @bermudezmt! In the version 4.x.x of twilio-node we [removed support for a WorkerSid for Cumulative and Real-Time Workers Statistics](https://github.com/twilio/twilio-node/pull/820). We have documented this change in the [upgrade guide](https://github.com/twilio/twilio-node/blob/main/UPGRADE.md?plain=1#L46).

Comment (bermudezmt) [reply rest/taskrouter/statistics/workers/cumulative/example-1.4.x.js:11]: Thank you so much!

**Review by bermudezmt — APPROVED** [review]

**Pull request merged**